src/BusinessLogic/Main.java

```diff
@@ -16,7 +17,9 @@
 	public static void main(String[] args) 
 	{
 		DBConnect con = new DBConnect();
-		con.getFoodCategories();
+		Meal testMeal = createMeal();
+		System.out.println(testMeal.toString());
+		
```

Comment:
I think its best if we just leave line 19 only when merging to master, leave line 20 and 21 out since all they are really doing is your own tests